audbackend/core/backend.py

```diff
@@ -321,28 +321,71 @@ def latest_version(
 
     def _ls(
             self,
-            path: str,
-    ) -> typing.List:  # pragma: no cover
-        r"""List content of path."""
+            folder: str,
+    ) -> typing.List[typing.Tuple[str, str, str]]:  # pragma: no cover
+        r"""List all files under folder."""
         raise NotImplementedError()
 
     def ls(
             self,
-            path: str,
-    ) -> typing.List[str]:
-        r"""List content of path.
+            folder: str = '/',
+            *,
+            latest_version: bool = False,
+    ) -> typing.List[typing.Tuple[str, str, str]]:
+        r"""List all files under folder.
+
+        Returns a sorted list of tuples
+        with path, extension and version.
+        If file extension is not empty,
+        it starts with a dot.
```

Comment:
It's not obvious what `it` is referring to. Looking only at the sentence one would guess `sorted list of tuples`, but this makes no sense.

Reply:
It refers to `file extension`, but since we anyway plan to get rid of it in #58 I simply removed the sentence.